auto_nag/scripts/patch_closed_bug.py

```diff
@@ -0,0 +1,92 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from datetime import datetime, timedelta
+
+import pytz
+from dateutil import parser
+
+from auto_nag import utils
+from auto_nag.bzcleaner import BzCleaner
+
+COMMENT_BODY = "A patch has been attached on this bug, which was already closed. Filing a separate bug will ensure better tracking. If this was not by mistake and further action is needed, please alert the appropriate party."
+
+
+class PatchClosedBug(BzCleaner):
+    def __init__(self):
+        super(PatchClosedBug, self).__init__()
+        self.days_count = self.get_config("days_count", 2)
+        today = pytz.utc.localize(datetime.utcnow())
```

Comment:
nit: maybe you can use `lmdutils.get_date_ymd("today")` just for the consistency with other tools.

Reply:
Done in 4c8ac758f1ff5b934020760e5aa4457fd2d40e04